### PR TITLE
Add more note for streamLoad java sample

### PR DESCRIPTION
**Pull request by caiconghui**
modify note info for sample output and some issue when use streamLoad java sample

### CODE REVIEW

**Comment by caiconghui** [comment]
related to #2199

File: samples/stream_load/java/DorisStreamLoad.java [hunk 1]
```diff
@@ -56,15 +56,47 @@
  * 3 run this class, you should see the following output:
  *
  * {
+ *     "TxnId": 27,
+ *     "Label": "39c25a5c-7000-496e-a98e-348a264c81de"
```
Comment (imay) [inline samples/stream_load/java/DorisStreamLoad.java]: Is there two label in response?

File: samples/stream_load/java/DorisStreamLoad.java [hunk 2]
```diff
@@ -95,6 +125,7 @@ protected boolean isRedirectable(String method) {
             StringEntity entity = new StringEntity(content);
             put.setHeader(HttpHeaders.EXPECT, "100-continue");
             put.setHeader(HttpHeaders.AUTHORIZATION, basicAuthHeader(DORIS_USER, DORIS_PASSWORD));
+            put.setHeader("label", "39c25a5c-7000-496e-a98e-348a264c81de")
```
Comment (kangkaisen) [inline samples/stream_load/java/DorisStreamLoad.java]: Would better add a comment for this line. the `label ` header is optional, not necessary.

**Comment by kangkaisen** [comment]
+1. LGTM.

**Review by kangkaisen — DISMISSED** [review]

File: samples/stream_load/java/DorisStreamLoad.java [hunk 3]
```diff
@@ -56,15 +56,45 @@
  * 3 run this class, you should see the following output:
  *
  * {
+ *     "TxnId": 27,
+ *     "Label": "39c25a5c-7000-496e-a98e-348a264c81de"
```
Comment (imay) [inline samples/stream_load/java/DorisStreamLoad.java]:
```suggestion
 *     "Label": "39c25a5c-7000-496e-a98e-348a264c81de",
```

File: samples/stream_load/java/DorisStreamLoad.java [hunk 4]
```diff
@@ -56,15 +56,45 @@
...
+ *     at shaded.org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
+ *     at shaded.org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
+ *     at shaded.org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
+ *
+ *2 run this class more than once, the status code for http response is still ok, and you will see
+ *  the following output:
+ *
+ * {
+ *     "TxnId": -1,
+ *     "Label": "39c25a5c-7000-496e-a98e-348a264c81de"
```
Comment (imay) [inline samples/stream_load/java/DorisStreamLoad.java]:
```suggestion
 *     "Label": "39c25a5c-7000-496e-a98e-348a264c81de",
```

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**